### PR TITLE
Don't minifiy builds of develop through CI packaging

### PR DESCRIPTION
**Pull request by turt2live**
Fixes https://github.com/vector-im/riot-web/issues/11864

This uses an environment variable because the build script assumes you want a production build, but we don't for this particular script. To avoid having a mess of NPM scripts to worry about, we'll just pass a flag down.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -8,320 +8,327 @@ const webpack = require("webpack");
 let og_image_url = process.env.RIOT_OG_IMAGE_URL;
 if (!og_image_url) og_image_url = 'https://riot.im/app/themes/riot/img/logos/riot-im-logo-black-text.png';
 
-module.exports = (env, argv) => ({
-    entry: {
-        "bundle": "./src/vector/index.js",
-        "indexeddb-worker": "./src/vector/indexeddb-worker.js",
-        "mobileguide": "./src/vector/mobile_guide/index.js",
+module.exports = (env, argv) => {
```
Comment (turt2live) [inline webpack.config.js]: this diff is ugly, but what has happened is the `({...})` bit has turned into a real function so the `if` statement below can be included. The entire config object has tabbed 1 to the right to account for the new `return {...}`.

**Comment by foldleft** [comment]
I'm okay with this, but it looks like @dbkr already took a look so I'll leave it to him

**Comment by jryans** [comment]
I am also sad about `build:dev` going away, but as long as there's some route to achieve the same, it should be fine.

**Comment by turt2live** [comment]
the trouble with `build:dev` is that it wasn't really worth having 4x the npm scripts which varied by only 1 or 2 flags.

**Pull request merged**